packages/service-library/src/servicelib/file_utils.py

```diff
@@ -4,6 +4,9 @@
 
 async def remove_directory(path: Path, only_children: bool = False) -> None:
     """Optional parameter allows to remove all children and keep directory"""
+    if not path.exists():
+        return
+
```

Comment:
Here to be honnest I would go same as with all python functions.
raise an exception if the path does not exist, and provide as argument ```missing_ok = False``` see [here](https://docs.python.org/3/library/pathlib.html#pathlib.Path.unlink)

Reply:
Fully agree with @sanderegg  as well on this. Please keep in mind my input above and reuse style and functionality when possible. It helps enormously maintainability